crates/iota-graphql-rpc/src/types/transaction_block_kind/programmable.rs

```diff
@@ -437,3 +445,19 @@ impl From<IotaArgument> for TransactionArgument {
         }
     }
 }
+
+// impl From<IotaArgumentV2> for TransactionArgument {
+//     fn from(argument: IotaArgumentV2) -> Self {
+//         use IotaArgumentV2 as S;
+//         use TransactionArgument as A;
+//         match argument {
+//             S::GasCoin => A::GasCoin(GasCoin { dummy: None }),
+//             S::Input(ix) => A::Input(Input { ix }),
+//             S::Result(cmd) => A::Result(TxResult { cmd, ix: None }),
+//             S::NestedResult(cmd, ix) => A::Result(TxResult { cmd, ix:
+// Some(ix) }),             S::Pure(v) => A::Pure(PureArg {
+//                 value: format!("{v:?}"),
+//             }),
+//         }
+//     }
+// }
```

Comment:
👀

Reply:
Forgot to remove it, did it now in https://github.com/iotaledger/iota/pull/8739/commits/9e14b4e95677439fcac01073f246a7e29ad7d4cf